src/sigstore.test.ts

```diff
@@ -24,9 +24,30 @@ describe('sign', () => {
   const payload = Buffer.from('Hello, world!');
 
   // Signer output
-  const signedPayload = {
-    base64Signature: 'signature',
-    cert: 'cert',
+  const signedPayload: Bundle = {
+    mediaType: 'application/vnd.dev.sigstore.bundle+json;version=0.1',
+    content: {
+      $case: 'messageSignature',
```

Comment:
TIL: This format output format is set by configuring `ts-proto` with `oneof=unions ` https://github.com/stephenh/ts-proto#oneof-handling

So this is an implementation detail of `ts-proto`, e.g. using buf's `protobuf-es` https://github.com/bufbuild/protobuf-es/blob/main/docs/generated_code.md#oneof-groups yields a slightly different output.